neqo-transport/src/packet.rs

```diff
@@ -62,6 +63,41 @@ impl PacketType {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum QuicVersion {
+    Draft27,
+    Draft28,
+}
+
+impl QuicVersion {
+    pub fn as_u32(self) -> Version {
+        match self {
+            Self::Draft27 => 0xff00_0000 + 27,
+            Self::Draft28 => 0xff00_0000 + 28,
+        }
+    }
+}
+
+impl Default for QuicVersion {
+    fn default() -> Self {
+        Self::Draft27
```

Comment:
Why 27?

Reply:
see above, 28 server causes tests to fail.